app/renderer/js/pages/preference/find-accounts.js

```diff
@@ -0,0 +1,79 @@
+'use-strict';
+
+const { remote, ipcRenderer } = require('electron');
+
+const { dialog } = remote;
+
+const BaseComponent = require(__dirname + '/../../components/base.js');
+
+class FindAccounts extends BaseComponent {
+	constructor(props) {
+		super();
+		this.props = props;
+	}
+
+	template() {
+		return `
+			<div class="settings-card certificate-card">
+				<div class="certificate-input">
+					<div>Organization URL</div>
+					<input class="setting-input-value" value="zulipchat.com"/>
+				</div>
+				<div class="certificate-input">
+					<button class="green w-150" id="find-accounts-button">Find accounts</button>
+				</div>
+			</div>
+		`;
+	}
+
+	init() {
+		this.$findAccounts = this.generateNodeFromTemplate(this.template());
+		this.props.$root.appendChild(this.$findAccounts);
+		this.$findAccountsButton = this.$findAccounts.querySelector('#find-accounts-button');
+		this.$serverUrlField = this.$findAccounts.querySelectorAll('input.setting-input-value')[0];
+		this.initListeners();
+	}
+
+	findAccounts(url) {
+		if (!url) {
+			return;
+		}
+		if (!url.includes(':')) {
+			url = 'https://' + url;
+		}
+		ipcRenderer.send('find-accounts', url + '/accounts/find');
+		ipcRenderer.on('find-accounts-reply', (event, result) => {
```

Comment:
What's the use of this?

Reply:
In `index.js`, we check if the app was able to open the URL in the browser. This happens in the following way:

```javascript
ipcMain.on('find-accounts', (event, url) => {
	const isBrowserOpened = electron.shell.openExternal(url);
	event.sender.send('find-accounts-reply', isBrowserOpened);
});
```
This code block is to notify the user if the URL entered was right or not.

Reply:
`'Check the organization URL and try again.'` is not a relevant error message here. 
`isBrowserOpened` would be false only if there wasn't an application was available to open the URL. I think we should avoid this altogether.